### PR TITLE
Docs: Clarinet troubleshooting guides - Added a step for Clarinet integrate command

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
Added troubleshooting step for `clarinet integrate` command. 

Files changed:
- docs/clarinet/troubleshooting.md

### CODE REVIEW

File: docs/clarinet/troubleshooting.md [hunk 1]
```diff
@@ -26,3 +26,13 @@ To manually add the directory to your path:
 6. Restart your shell/VSCode.
 
 If you did not install Clarinet to your default directory, you would need to modify the path so that Clarinet points to the correct directory. 
+
+## I am unable to start Devnet though my Docker is running.
+
+When you run the `Clarinet integrate` command, you might experience the error: "unable to start Devnet: make sure that Docker is installed on this machine and running."
+
+You can resolve the issue by creating a symlink of the docker.sock file. To do that, In your terminal, navigate to your Clarinet project directory and run the following command:
```
Comment (trichards-hiro) [inline docs/clarinet/troubleshooting.md:34]:
```suggestion
You can resolve the issue by creating a symlink of the `docker.sock` file. To create this symlink, in your terminal, navigate to your Clarinet project directory and run the following command:
```

File: docs/clarinet/troubleshooting.md [hunk 2]
```diff
@@ -26,3 +26,13 @@ To manually add the directory to your path:
...
+
+`sudo ln -s /Users/<your-username>/.docker/run/docker.sock /var/run/docker.sock` 
+
+Now, run the command `clarinet integrate` to see the Devnet up and run.
```
Comment (trichards-hiro) [inline docs/clarinet/troubleshooting.md:38]:
```suggestion
Now, run the command `clarinet integrate` to see the Devnet up and running.
```

**Review by trichards-hiro — APPROVED** [review]
Just a few minor suggestions. Nothing major. Looks good!

**Comment by LakshmiLavanyaKasturi** [comment]
@lgalabru Can you please review my changes? Thanks!

**Review by lgalabru — APPROVED** [review]
LGTM, thanks @LakshmiLavanyaKasturi!

**Pull request merged**